### ISSUE TITLE
PT001 implementation from ruff is opposite of PT001 from flake8-pytest-style

### ISSUE THREAD

**Issue by ssbarnea** [charliermarsh/ruff#3956]
Apparently ruff implementation of PT001 is the exact opposite of flake8 one. Ruff auto-fix is breaking flake8 rule. See https://github.com/m-burst/flake8-pytest-style/blob/master/docs/rules/PT001.md

```
PT001 use @pytest.fixture over @pytest.fixture()
```

What should we do? Can we agree on a recommendation and modify either ruff or flake8-pytest-style default settings to avoid this conflict?

Related:
- https://github.com/m-burst/flake8-pytest-style/issues/4

**Comment by ssbarnea** [comment]
Oops, I discovered that was caused by `pytest-fixture-no-parentheses = true` being inside my flake8 conflict, so a false bug report. I removed the customization and now they match.
